torch/csrc/jit/codegen/cuda/mutator.cpp

```diff
@@ -212,17 +212,7 @@ Statement* OptOutMutator::mutate(ReductionOp* rop) {
 }
 
 Statement* OptOutMutator::mutate(kir::GridReduction* gr) {
```

Comment:
I assume this means OptOutMutator is just a dispatch that can return a statement? Do we want to either do all of these at the same time or leave them as is?

Reply:
These are just placeholders, anticipating that we're not going to use/need OptOutMutator for Kernel IR nodes. 

What do you think?